app/client/src/ce/navigation/FocusStrategy/NoIDEFocusStrategy.ts

```
@@ -8,8 +8,11 @@ export const NoIDEFocusStrategy: FocusStrategy = {
   *getEntitiesForStore() {
     return [];
   },
-  getEntityParentUrl(): string {
+  getEntityParentUrl: (): string => {
     return "";
   },
+  getUrlKey: function* (url: string) {
+    return url;
+  },
```

Biome (1.9.4)

[error] This generator function doesn't contain yield.



(lint/correctness/useYield)